iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/AttachNewLineOnLine.java

```diff
@@ -54,7 +54,9 @@ public class AttachNewLineOnLine implements NetworkModification {
      * line2Id is line.getId() + "_2". <br>
      *
      * @see #AttachNewLineOnLine(double, String, String, String, String, String, Line, LineAdder)
+     * @deprecated Use {@link AttachNewLineOnLineBuilder} instead.
      */
+    @Deprecated(since = "4.10.0")
```

Comment:
Just wondering if deprecated is the appropriate term / tag, as we will just change the access to protected (and not really delete it one day).

Reply:
Not sure if this is appropriate indeed! I assumed this javadoc was only useful from a user standpoint (not a developer one) but maybe I am wrong. What do you suggest?

Reply:
I deprecated all constructors except for those used in the builders and just added in the javadoc that they will eventually be package private. Do you agree?